mdpow/fep.py

```diff
@@ -1127,6 +1127,37 @@ class Goct(Gsolv):
                  }
 
 
+class Gwoct(Gsolv):
```

Comment:
Can you derive from `Goct` instead? Then just override `solvent_default` and `dirname_default`. `schedules` can be taken from the parent class.